lib/common/remote.c

```diff
@@ -34,21 +34,7 @@
 #include <crm/common/remote_internal.h>
 
 #ifdef HAVE_GNUTLS_GNUTLS_H
-#  undef KEYFILE
 #  include <gnutls/gnutls.h>
-
-const int psk_tls_kx_order[] = {
-    GNUTLS_KX_DHE_PSK,
-    GNUTLS_KX_PSK,
-};
-
-const int anon_tls_kx_order[] = {
-    GNUTLS_KX_ANON_DH,
-    GNUTLS_KX_DHE_RSA,
-    GNUTLS_KX_DHE_DSS,
-    GNUTLS_KX_RSA,
-    0
-};
```

Comment:
Unless I'm missing something, doesn't this mean the block inside the ifdef is now empty so it and the endif can go away too?

Reply:
It still is used for the include of gnutls.h, which is needed for other ifdef'd blocks later in the file.